File: development/OnBoarding/OnBoarding/OnBoarding.Contract/Associate/Associate.cs
namespace OnBoarding.Contract
{
    using System;

    public class Associate
    {
        public Guid Id { get; set; }

        public string Name { get; set; }

        public string Code { get; set; }

        public Guid RoleId { get; set; }

        public Guid ProjectId { get; set; }
    }
}

File: development/OnBoarding/OnBoarding/OnBoarding.Web/src/app/model/associate.ts
export interface IAssociate {
    Id: string,
    Code: string,
    Name: string,
    RoleId: string,
    ProjectId: string
}

export class Associate implements IAssociate {

    constructor(
        public Id: string,
        public Code: string,
        public Name: string,
        public RoleId: string,
        public ProjectId: string) {
    }
}